Kernel/FileSystem/Ext2FS/ext2_types.h

```diff
@@ -15,40 +15,16 @@
 #    define _EXT2_TYPES_H
 
 #    define __S8_TYPEDEF __signed__ char
-#    define __U8_TYPEDEF unsigned char
 #    define __S16_TYPEDEF __signed__ short
-#    define __U16_TYPEDEF unsigned short
 #    define __S32_TYPEDEF __signed__ int
-#    define __U32_TYPEDEF unsigned int
 #    define __S64_TYPEDEF __signed__ long long
-#    define __U64_TYPEDEF unsigned long long
-
-#    ifdef __U8_TYPEDEF
-typedef __U8_TYPEDEF __u8;
-#    else
-typedef unsigned char __u8;
-#    endif
 
 #    ifdef __S8_TYPEDEF
 typedef __S8_TYPEDEF __s8;
 #    else
 typedef signed char __s8;
 #    endif
 
-#    ifdef __U16_TYPEDEF
```

Comment:
This file is third party and a different license. We shouldn't be modifying it like this

Reply:
Dropped the commit.